sensor_msgs/Image ros_image
int64 x
int64 y
int64 w
int64 h
